docs/setupandinstallation.md

```diff
@@ -200,12 +199,6 @@ This section describes how to validate your back end installation, and then star
 
 The following are known issues with the current release of Spartacus JavaScript Storefront:
 
-- When using SAP Commerce 1808 for your back end, you are currently not able to add payment details or address details in the Spartacus storefront, which prevents successful checkout. However, if you add payment and address details through the Accelerator electronics storefront, they will then appear in the Spartacus storefront, and you will be able to check out.
-
-- The Spartacus storefront is currently missing all categories.
```

Comment:
I removed this and issue below since they do show up if using the `build.gradle` file, should I keep them instead?